src/file_process.cpp

```diff
@@ -0,0 +1,180 @@
+/** \brief This source file defines functions for Output File Processing.
+*
+* The output file generated by the simulator is processed
+* to produce a more readable format. The data is stored in
+* output file *_proc.txt in a table format as shown below:
+*
+*Time           value  port                component
```

Comment:
Spacing after the * is required as per the format that you are following

(* Time) rather than *Time
Same applies for below few lines

Reply:
solved